components/articles/articles-page.jsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react"
+import { getsAllArticles } from "../../utils/api"
+import ArticleCards from "./article-card"
+
+
+const Articles = () =>{
+    const [articles, setArticles] = useState([])
+    useEffect(()=>{
+        getsAllArticles().then(({articles})=>{
+            setArticles(articles)
+        })
```

Comment:
Please add some isLoading functionality so that people know that the articles are still being fetched.